portal/docker.sbt

```diff
@@ -1,21 +1,3 @@
-defaultLinuxInstallLocation in Docker := "/cmwl_pipeline/app"
-packageName in Docker := "cmwl_pipeline"
-dockerExposedVolumes := Seq("/cmwl_pipeline/app/logs")
-dockerExposedPorts := Seq(8080)
```

Comment:
When I run `sbt clean docker:publishLocal` it warns me
```
[warn] [1] There are no exposed ports for your docker image
[warn]  Configure the `dockerExposedPorts` or `dockerExposedUdpPorts` setting. E.g.
[warn] 
[warn]  // standard tcp ports
[warn]  dockerExposedPorts ++= Seq(9000, 9001)
[warn] 
[warn]  // for udp ports
[warn]  dockerExposedUdpPorts += 4444
```
Shouldn't we keep ports settings here?

Reply:
Added exposed ports. I'm not sure about safety explicit cast to Int, but I think it's clear that the port should be Int in .env file. Do I need to add type checking?

Reply:
I don't think we have a way to recover from such cases so it should be ok.